app/common/directives/add-category.directive.js

```diff
@@ -0,0 +1,69 @@
+module.exports = AddLabelDirective;
+
+AddLabelDirective.$inject = [];
+
+function AddLabelDirective() {
```

Comment:
Can we call this category for consistency, since we're not using label elsewhere?